core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceFactory.java

```diff
@@ -166,11 +170,26 @@ public static List<AnnotatedServiceElement> find(
                       .collect(toImmutableList());
     }
 
-    private static HttpStatus defaultResponseStatus(Method method) {
+    private static HttpStatus defaultResponseStatus(Method method, Class<?> clazz) {
         final StatusCode statusCodeAnnotation = AnnotationUtil.findFirst(method, StatusCode.class);
         if (statusCodeAnnotation == null) {
```

Comment:
nit: Let's do the early return because this block gets bigger.
```java
if (statusCodeAnnotation != null) {
    ...
}
...
```

Reply:
Thanks! Should've read this more carefully 😆.
https://armeria.dev/community/developer-guide#prefer-early-return-style